include/HDF5_Typedef.h

```diff
@@ -560,6 +560,7 @@ struct InputPara_t
    double GREP_LogBinRatio;
    double GREP_MaxRadius;
    double GREP_MinBinSize;
+   int    GREP_Opt_FixUp;
```

Comment:
is this better use `bool`?

Reply:
No, in GAMER all `bool` runtime variables are in type of `int` for dumping the HDF5 file.